catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -385,7 +350,7 @@
                         <configuration>
                             <rules>
                                 <ArtifactSizeEnforcerRule implementation="org.codice.maven.artifactsize.ArtifactSizeEnforcerRule">
-                                    <maxArtifactSize>63.2_MB</maxArtifactSize>
+                                    <maxArtifactSize>10.5_MB</maxArtifactSize>
```

Comment:
Currently the size is not reduced since the embedded dependencies are in the new Solr cache module.  With some cleanup, I think we could get to a point where there is one service used by both the provider and the cache to avoid this duplication but I did not do that as a part of this PR.

Reply:
That's fine - we probably don't have to take it that far since most of the benefits have been obtained, although we certainly could - the point is that if we **don't** install Solr Cache support then we **will** see the size reduction because those bundles with the additional embedded dependencies won't be installed, which means we don't have to ship them. Does that sound right? 

Not shipping them might just require a little features file rework instead of writing a new service.

Reply:
That sounds right.